### PR TITLE
Updates the MacOSX demo tutorial to clarify CLI commands 

### PR DESCRIPTION
**Pull request by betarelease**
- uses the latest CLI options

## Description

Fixes pyrsia documentation on the website.

This PR updates the MacOSX demo tutorial to clarify CLI commands ands also uses the latest version of CLI commands. This is only a documentation update

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [ ] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [x] I've requested a review  from `pyrsia/collaborators`.

I have not linked this to any issue. This PR should be treated as fixing documentation for clarity and readability.

### CODE REVIEW

**Comment by prince-chrismc** [comment]
😡 https://github.community/t/annotation-limitation/17998/2

So the linting tool does not show the bulk of warnings under "files changed" so we need to go to details to see all of them 🤢  not easier

File: docs/tutorials/setup_on_macosx.md [hunk 1]
```diff
@@ -3,11 +3,15 @@
 Download a fresh copy of the codebase by cloning the repo or updating to the HEAD of `main`
 Let's call this folder `PYRSIA_HOME`. We will refer to this name in the following steps.
 
-Build binaries for `pyrsia_node` by running `cargo build --workspace`.
+Build binaries for `pyrsia_node` by running
```
Comment (prince-chrismc) [inline docs/tutorials/setup_on_macosx.md:6]:
```suggestion
Build binaries for `pyrsia_node` by running:

```

File: docs/tutorials/setup_on_macosx.md [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
+```
```
Comment (prince-chrismc) [inline docs/tutorials/setup_on_macosx.md:7]:
```suggestion
```sh
```

File: docs/tutorials/setup_on_macosx.md [hunk 3]
```diff
@@ -68,12 +74,48 @@ Also that means we did not break anything 😜
 
 You should now be able to interact with the Pyrsia Node.
 
+### Pyrsia CLI options
```
Comment (prince-chrismc) [inline docs/tutorials/setup_on_macosx.md:77]:
```suggestion
### Pyrsia CLI options

```

**Review by prince-chrismc — COMMENTED** [review]
Some nitty whitespace

File: docs/tutorials/setup_on_macosx.md [hunk 4]
```diff
@@ -168,3 +210,8 @@ Now node B is acting as the pull-through cache and should show a line similar to
 ```
 # DEBUG pyrsia::docker::v2::handlers::blobs> Step 2: "sha256:3d243047344378e9b7136d552d48feb7ea8b6fe14ce0990e0cc011d5e369626a" successfully stored locally from Pyrsia network.
 ```
+Success!!!
+
+You have just built yourself a working Pyrsia network. Enjoy using it and showcasing it to your teams and please share any feedback!
+
+Next you can, follow the instructions in [demo.md](demo.md) and setup a real Pyrsia network and use it with your CI system.
```
Comment (prince-chrismc) [inline docs/tutorials/setup_on_macosx.md:217]: This link wont work on the website 🙃

**Comment by prince-chrismc** [comment]
In #699 I called out a few section that can not be copy pasted... I think it makes it a ton easier instead of having to **carefully** select the write command

It's also unclear with the combination of commands + outputs... the convention is to use the `$` before the command https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output

File: docs/tutorials/setup_on_macosx.md (hunk 4 above)
Comment (betarelease) [reply docs/tutorials/setup_on_macosx.md]: Updated the url in the new changes.

**Comment by betarelease** [comment]
Also fixed the markdown lint issues with copypaste code.

**Review by erwin1 — APPROVED** [review]

**Pull request merged**